gradle.properties

```diff
@@ -23,3 +23,4 @@ libVcsUrl=https://github.com/mozilla-mobile/android-components.git
 
 libLicense=MPL-2.0
 libLicenseUrl=https://www.mozilla.org/en-US/MPL/2.0/
+android.enableUnitTestBinaryResources=true
```

Comment:
This isn't necessary, right?

Reply:
I did it because Robolectric was filling the screen with complaints about it missing. It's not strictly necessary though.